sw/device/lib/dif/dif_base.h

```diff
@@ -128,6 +131,27 @@ inline dif_toggle_t dif_bool_to_toggle(bool val) {
   return val ? kDifToggleEnabled : kDifToggleDisabled;
 }
 
+/**
+ * Converts a multi-bit bool to a `dif_toggle_t`.
+ *
+ * @param val A multi-bit bool value.
+ * @return Corresponding dif_toggle_t value.
+ */
+inline dif_toggle_t dif_multi_bit_bool_to_toggle(multi_bit_bool_t val) {
+  switch (val) {
+    case kMultiBitBool4True:
+      FALLTHROUGH_INTENDED;
```

Comment:
Do you need this here? IME clang's -Wfallthrough only requires annotation when the body of the case is nonempty.

Reply:
Fixed.